pkg/localstorev2/reserve.go

```diff
@@ -92,6 +92,8 @@ func (db *DB) IsWithinStorageRadius(addr swarm.Address) bool {
 	return swarm.Proximity(addr.Bytes(), db.baseAddr.Bytes()) >= db.reserve.Radius()
 }
 
+func (db *DB) IsFullySynced() bool { return false }
```

Comment:
was this left out temporarily on purpose?
this function should check that the reserve size is above the threshold and the chunk rate is zero

Reply:
Yes I havent implemented it. I just added it to satisfy the interfaces.

Reply:
What I had in mind for this was to have a simple function created in node.go that would be passed to the storage incentives agent that would check that the puller rate is zero and the reserve size is above the threshold, this way, the localstore does not have to implement the `IsFullySynced`

Reply:
Do you want me to do this in the current PR or should we do it in a separate one?

Reply:
So for now I have kept this in the storer as we already have the syncer and size values here. So this function is a one-liner either way. We can move it later if you still feel strongly about this.